public/game/views/game-view.coffee

```diff
@@ -42,7 +38,13 @@ GameView = (context) ->
 							nxt.Binding context.selector_input_disabled, (disabled) ->
 								nxt.Attr 'disabled' if disabled
 
-				TimespanView context.roundTimerViewModel
+				nxt.Binding context.selector_input_disabled, (isDisabled) ->
```

Comment:
Let's rename this cell as discussed during RSConf Hackathon :cat: 
Something like `puzzle_solved`?